nuxeo-runtime/nuxeo-stream/src/test/java/org/nuxeo/lib/stream/tests/log/TestLogKafka.java

```diff
@@ -150,8 +149,8 @@ public void testSeekByTimestamp() throws Exception {
         appender.append(0, KeyValueMessage.of("id5"));
 
         //Get the time and wait 3 seconds
+        Thread.sleep(3000L);
```

Comment:
Not sure why you specify `3000L` instead of just `3000`, the int will be promoted to long automatically. Is that an IDE style thing?

Reply:
I thought, in a previous review you said longs should be with L.

Reply:
I don't remember that...

But I think somebody else mentioned the fact that if you _have_ to express such a long then it should be with an uppercase `L` and not a lowercase `l` which would be confused with a `1` (one).